samtranslator/model/sam_resources.py

```diff
@@ -2422,6 +2429,26 @@ def _construct_appsync_schema(
 
         return schema
 
+    def _construct_appsync_api_keys(
+        self, api_keys: Dict[str, aws_serverless_graphqlapi.ApiKey], api_id: Intrinsicable[str]
+    ) -> List[Resource]:
+        resources: List[Resource] = []
+
+        # TODO: Add datetime parsing for ExpiresOn
```

Comment:
Why?

Reply:
That's what we're planning to add because currently they support Double only which is awful UX

Reply:
Will that be confusing? I'm assuming you'd still support doubles as well?

Reply:
Yes, we will still have support for double (which is epoch time representation).